src/review_heatmap/libaddon/gui/basic/interface.py

```diff
@@ -45,7 +45,11 @@
 from __future__ import (absolute_import, division,
                         print_function, unicode_literals)
 
-from collections import MutableSequence, MutableSet, MutableMapping
+try:
+    from collections.abc import MutableSequence, MutableSet, MutableMapping  # noqa
+except ImportError:
+    from collections import MutableMapping, MutableSet, MutableMapping  # noqa
```

Comment:
Are you sure to import twice `MutableMapping`?

Reply:
```suggestion
    from collections import MutableSequence, MutableSet, MutableMapping  # noqa
```